index.spec.js

```diff
@@ -0,0 +1,46 @@
+const request = require('supertest');
+const server = require('./api/server');
+const db = require('./dbConfig');
+
+afterAll(() => db('games').truncate());
+
+describe('server.js', () => {
+  describe('/games endpoint POST', () => {
+    it('should return 201 status code', async () => {
+      const game = {
+        title: 'FIFA-19',
+        genre: 'Sports'
+      };
+      const response = await request(server)
+        .post('/games')
+        .send(game);
+      expect(response.status).toBe(201);
+    });
+    it('should return status 422 when missing genre', async () => {
+      const game = {
+        title: 'FIFA-19',
+        releaseYear: 2019
+      };
+      const response = await request(server)
+        .post('/games')
+        .send(game);
+      expect(response.status).toBe(422);
+    });
+    it('should return status 422 when missing title', async () => {
+      const game = {
+        genre: 'Sports'
+      };
+      const response = await request(server)
+        .post('/games')
+        .send(game);
```

Comment:
Since you're making this same request, just with a different `game` object used for `send` each time, why not make a helper function to make your code shorter and perhaps even more readable? For example:
```javascript
describe('server.js', () => {
  describe('/games endpoint POST', () => {
    const post = async game => await request(server)
      .post('/games')
      .send(game);

    it('should return 201 status code', () => {
      const game = {
        title: 'FIFA-19',
        genre: 'Sports'
      };
      const response = post(game);
      expect(response.status).toBe(201);
    });

    // ...
  });
});
```
I'd also consider using object destructuring more, and perhaps reducing some of the steps with extra variables with our new helper function:
```javascript
it('should return 201 status code', () => {
  const { status } = post({
    title: 'FIFA-19',
    genre: 'Sports'
  });
  expect(status).toBe(201);
});
```